src/modules/staking/services/staking.compute.service.ts

```diff
@@ -316,4 +325,23 @@ export class StakingComputeService {
             minutes: Math.floor(frequencyMinutes),
         });
     }
+
+    @ErrorLoggerAsync({
+        logArgs: true,
+    })
+    @GetOrSetCache({
+        baseKey: 'stake',
+        remoteTtl: CacheTtlInfo.ContractState.remoteTtl,
+        localTtl: CacheTtlInfo.ContractState.localTtl,
+    })
+    async deployedAt(stakeAddress: string): Promise<number> {
```

Comment:
You can add this as a field into staking farm model and expose it

Reply:
done